cmd/translate/main_rest_integration_test.go

```diff
@@ -28,7 +28,15 @@ func TestMain(m *testing.M) {
 
 	conn.Close()
 
-	os.Exit(m.Run())
+	code := m.Run()
+
+	// To avoid tp.ForceFlush() from indefinitely blocking, allocate 5 seconds for exporting spans.
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Do not use timeout context, tests already have a timeout (30 seconds by default). Use **context.Background()**.